src/init3d.F

```diff
@@ -407,7 +407,7 @@ subroutine init3d(xh,rxh,uh,ruh,xf,rxf,uf,ruf,yh,vh,rvh,yf,vf,rvf,  &
         if(dowr) write(outfile,*)
         if(dowr) write(outfile,*) '  Parcels ! '
         if(dowr) write(outfile,*) '  npvals,nparcelsLocal = ',npvals,nparcelsLocal
-        if(dowr) write(outfile,*) '  Initial parcel locations (x,y,z):'
+        !if(dowr) write(outfile,*) '  Initial parcel locations (x,y,z):'
```

Comment:
remove this line if it is no longer needed.

Reply:
Fixed